modules/api/src/main/java/org/apache/ignite/schema/definition/ColumnDefinition.java

```diff
@@ -43,10 +43,9 @@ public interface ColumnDefinition {
     boolean nullable();
 
     /**
-     * Returns column default value expression.
+     * Returns default value definition.
      *
-     * @return Default column value expression.
+     * @return Default value definition.
      */
-    //TODO: IGNITE-14479: Rename to defaultValueExpr or allow constants? How to distinct expression from string constant???
-    Object defaultValue();
+    DefaultValueDefinition defaultValueDefinition();
```

Comment:
```suggestion
<T extends DefaultValueDefinition> T defaultValueDefinition();
```

WDYT, will be better to use such approach here?

Reply:
Oh, I've got the idea: to avoid manual casting every time. Make sense actually

Reply:
fixed